CHANGELOG_CLOUD.md

```diff
@@ -0,0 +1,688 @@
+# Changelog
+All notable changes to this project for *our cloud releases* will be documented in this file.
+The v23.0.0 release combines the open source and cloud releases and we won't have any
```

Comment:
Does this need to be declared here? Since cloud backend visibility is limited, I don't think we should add this here.

Reply:
why not?